cds-snc/report-a-cybercrime
# submitReport graphQL mutation

resolves #340 

also got rid of the "flagging" related code (from when we were submitting a single ip/site/email in Concept 1 testing)

To test in GraphiQL:
- start up arango and minio
- run `node index.js` in api dir
- go to `localhost:3000`

variables:
```
{
  "scamInfo": {
    "howWereYouContacted": ["phone", "email"], 
    "whenWereYouContacted": "today"
  },
  "contactInfo": {
    "contactInfoName": "Steve",
    "contactInfoEmail": "steve@go.com",
    "contactInfoPhone": "555-5555"
  },
  "lostMoney": {
    "lostAmount": "100,000",
    "lostCurrency": "CAD",
    "lostMethodsOfPayment": ["cash", "other"],
    "lostOtherMethodOfPayment": "catcoin"
  }
}
```

run:
```
mutation submitReport($scamInfo: scamInfoInput!, $lostMoney: lostMoneyInput!, $contactInfo: contactInfoInput!) {
  submitReport(scamInfo: $scamInfo, lostMoney:$lostMoney, files: ["file1.txt", "screenshot.jpg"], contactInfo: $contactInfo) {
    reportID
  }
}
```

a uuid should be returned, and a corresponding record added to arango
